notebooks/tutorial/8 - Getting tomes from the ADX.ipynb

```diff
@@ -0,0 +1,230 @@
+    }
+   },
+   "cell_type": "markdown",
+   "id": "9ac3586b",
+   "metadata": {},
+   "source": [
+    "![adx_tome_explainer.png](attachment:adx_tome_explainer.png)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9db8f974",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dataset_id = \"c5943d0f4bbce90b2f7e5f8488595e86\"\n",
+    "bucket = None \n",
+    "prefix = None"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2e805cf6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "if dataset_id == \"c5943d0f4bbce90b2f7e5f8488595e86\":\n",
+    "    raise Exception(\"Your dataset_id is still the default. Please subscribe on the ADX\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f6715957",
+   "metadata": {},
+   "source": [
+    "# Download the tomes\n",
+    "\n",
+    "Since the tomes are only about 15 GB the cost is very little to transfer all of them. Therefore we will simply transfer them all. Check the [previous notebook]() for a cost calculator."
```

Comment:
```suggestion
    "Since the tomes are only about 15 GB the cost is minimal to transfer all of them, so we will export them all. Check the [previous notebook]() for a cost calculator."
```